Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  * Each data collection should only have one specimen type
+  * Please choose the lowest-level specimen type that is appropriate for your collection
+  * If you believe that there is more than one specimen type, you might need a new data collection
+  * If you cannot find an appropriate specimen type, or you are not sure which one to use,
+    please email [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+    and we will add a new specimen type or help you choose the appropriate one
+    
+#### **is data about** - *species*
+  * This is the common name of the species that this data collection is about
+  * This field is controlled by the vocabulary in the `species` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one species associated with a data collection
+  * Data collections about human subjects should be separate from other species
+  
+#### **is data of type** - *modality*
+  * This is the name of the data modality for this collection
+  * This field is controlled by the vocabulary in the `modality` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one modality associated with a data collection
+  
+#### **is subject of resource** - *URL*
+  * This is usually a link to the repository where the raw data is stored.
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+
+#### **was collected using** - *technique*
+  * This is the name technique used to acquire data for this collection
+  * This field is controlled by the vocabulary in the `technique` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one technique associated with a data collection
+  
+#### **Title** - *string*
+  * This is a simple string is currently not displayed. 
+    (note: Title _is_ displayed for _data collection projects_)
+    
+#### **Short title** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection title
+    (note: Short title _is not_ displayed for _data collection projects_)
+  * Please limit short titles to 30 characters or less
+  
+#### **Description** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection description
+  
+#### **Access control** - *access control*
+  * This indicates the level of control of access to these data
+  * This field is controlled by the vocabulary in the `access control` column on `values` sheet
+  * Most non-human data collections should be `open`
+  * Human data collections should be `controlled`
```

Comment:
Not true. please amend to:

"Human data collections may be either open or controlled access, depending on the type of data and your IRB approval."

Reply:
Fixed